src/lib/requests/baseRequests.js

```diff
@@ -269,7 +271,13 @@ class BaseRequests {
      */
     _buildHeaders(method, resourceType, dest) {
         let headers = {
-            'content-type': `application/vnd.interoperability.${resourceType}+json;version=1.0`,
+            'content-type': `application/vnd.interoperability.${resourceType}+json;version=${
```

Comment:
I'm not a big fan of these 4 line long `&&` statements - could we adopt a similar approach to `this.resourceEndpoints` where we set the defaults in the constructor and then we can just call `this.resourceVersions[resourceType].contentVersion`

That also has the bonus of explicitly defining the different resourceTypes, and would make it easy to spot errors and make changes in the future.

Reply:
@lewisdaly thanks for the proposal :) looks much nicer..
I haven't separated the defaults in another module, though, because it doesn't need some kind of processing like the resourceEndpoints, for example. Would love to get your review on the current PR.